client/my-sites/plugins/search-categories/index.tsx

```diff
@@ -0,0 +1,166 @@
+import page from '@automattic/calypso-router';
+import Search, { ImperativeHandle } from '@automattic/search';
+import { isDesktop } from '@automattic/viewport';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { FC, useCallback, MutableRefObject, useRef, RefObject, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import ScrollableHorizontalNavigation from 'calypso/components/scrollable-horizontal-navigation';
+import { setQueryArgs } from 'calypso/lib/query-args';
+import scrollTo from 'calypso/lib/scroll-to';
+import withDimensions from 'calypso/lib/with-dimensions';
+import Categories from 'calypso/my-sites/plugins/categories';
+import {
+	ALLOWED_CATEGORIES,
+	useCategories,
+} from 'calypso/my-sites/plugins/categories/use-categories';
+import { useGetCategoryUrl } from 'calypso/my-sites/plugins/categories/use-get-category-url';
+import { useLocalizedPlugins } from 'calypso/my-sites/plugins/utils';
+import { recordTracksEvent, recordGoogleEvent } from 'calypso/state/analytics/actions';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { useTermsSuggestions } from './useTermsSuggestions';
+
+import './style.scss';
+
+const SearchBox: FC< {
```

Comment:
Component taken from the now removed `search-box-header` folder.

Reply:
What's preventing you using from the existing component?